src/sparsezoo/utils/helpers.py

```diff
@@ -14,9 +14,28 @@
 
 import glob
 import os
+from typing import Any
 
 
-__all__ = ["create_dirs", "create_parent_dirs", "clean_path", "remove_tar_duplicates"]
+__all__ = [
+    "create_dirs",
+    "create_parent_dirs",
+    "clean_path",
+    "remove_tar_duplicates",
+    "convert_to_bool",
+]
+
+
+def convert_to_bool(val: Any):
```

Comment:
Maybe add some unit tests for this one? Seems like it should be pretty straightforward to test